resources/sass/vendors/_index.scss

```diff
@@ -4,9 +4,11 @@
 @import 'vue-transitions';
 @import 'v-calendar';
 @import 'i-view-notice-custom';
+@import 'swiper-custom';
 
 // vendors from node_modules
 @import '~leaflet/dist/leaflet.css';
```

Comment:
i think its better to put lib files in the top of the file, this way it will be easier to redefine their own rules, because the styles in the bottom have bigger priority